src/featurestorebundle/feature/FeatureStore.py

```diff
@@ -45,12 +45,17 @@ def get_latest(
         lookback: Optional[str] = None,
         features: Optional[List[str]] = None,
         skip_incomplete_rows: bool = False,
+        include_spurious: bool = False,
     ) -> DataFrame:
         entity = self.__entity_getter.get_by_name(entity_name)
         feature_store = self.__features_reader.read(entity_name)
         features = features or self.__features_manager.get_registered_features(feature_store)
         metadata = self.get_metadata(entity_name, features)
-        feature_list = self.__feature_list_factory.create(metadata)
+        feature_list = self.__feature_list_factory.create(metadata, include_spurious=include_spurious)
+
```

Comment:
I think including or not including spurious features should be bussiness of either some FilteringManager or maybe some dedicated class

Reply:
I've moved the functionality to `FilteringManager`. It's however not the right class to put this functionality to. I don't know the original purpose of `FilteringManager`, but AFAIK its helping with adjusting the feature store *row-wise*. Keeping or Skipping spurious features is a column-wise operation and therefore it may not be a business of the `FilteringManager`.
I guess it's now up to you to say whether `FilteringManager` should be able to "filter" the result in both dimensions, or it should only be responsible for row-wise filtering.